tests/test-setup.sh

```diff
@@ -0,0 +1,52 @@
+set -eu
+
+# */
+# To use envtest is required etcd, kube-apiserver and kubetcl binaries in the testbin directory.
+# This sript will perform this setup for linux or mac os x envs.
+# */
+
+K8S_VER=v1.18.2
+ETCD_VER=v3.4.3
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+ARCH=$(uname -m | sed 's/x86_64/amd64/')
+ETCD_EXT="tar.gz"
+
+rm -rf testbin
+mkdir -p testbin
+
+# install etcd bin
+# the extension for linux env is not equals for mac os x
+if [ $OS == "darwin" ]; then
+   ETCD_EXT="zip"
+fi
+[[ -x testbin/etcd ]] || curl -L https://storage.googleapis.com/etcd/${ETCD_VER}/etcd-${ETCD_VER}-${OS}-${ARCH}.${ETCD_EXT} | tar zx -C testbin --strip-components=1 etcd-${ETCD_VER}-${OS}-${ARCH}/etcd
```

Comment:
https://storage.googleapis.com/etcd/v3.4.3/etcd-v3.4.3-darwin-amd64.tar.gz
`No such object: etcd/v3.4.3/etcd-v3.4.3-darwin-amd64.tar.gz`
what am I missing here? and since when do we support mac?

Reply:
I forgot to delete this file, deleting it now